package.json

```diff
@@ -20,6 +20,7 @@
   ],
   "dependencies": {
     "bootstrap": "^3.1.1",
+    "bootstrap4": "npm:bootstrap@^4.3.1",
```

Comment:
Can you remove this. I wouldn't want to introduce BS4 as a dependency for current users who are only using BS3.

Reply:
I can. But it'll break the demo in `run`. So I'd have to remove it as well.

Reply:
I moved it to the dev dependencies. How do you like that approach?